thoth-api/migrations/0.9.6/up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE work DROP CONSTRAINT work_doi_check;
+ALTER TABLE work ADD CONSTRAINT work_doi_check
+  CHECK (doi ~ '^https:\/\/doi\.org\/10\.\d{4,9}\/[-._\;\(\)\[\]\/:a-zA-Z0-9]+$');
```

Comment:
Should update `reference` `doi` and `institution` `institution_doi` checks similarly